src/beacon/app/main.py

```
@@ -0,0 +1,1370 @@
+    total_devices: int
+    active_devices: int
+    offline_devices: int
+    deployed_devices: int 
+    not_deployed: int
+    recalled_devices: int
+
+# Custom response to handle JSON encoding
+def create_json_response(content):
+    """Create a Response with properly encoded JSON content"""
+    json_content = json.dumps(content, cls=CustomJSONEncoder)
+    return Response(content=json_content, media_type="application/json")
```

Ruff (0.11.9)

Undefined name `Response`

(F821)